kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/utils/DeploymentUtils.java

```diff
@@ -235,6 +241,31 @@ public static void registryCredentialsSecret(String namespace) {
         }
     }
 
+    /**
+     * Collect cluster info.
+     *
+     * @param namespace the namespace
+     * @param testClassName the test class name
+     * @param testMethodName the test method name
+     */
+    public static void collectClusterInfo(String namespace, String testClassName, String testMethodName) {
+        String formattedDate = DateTimeFormatter.ofPattern("yyyy_MM_dd_HH_mm_ss")
```

Comment:
(non blocking) you could use DateTimeFormatter.ISO_INSTANT

Reply:
Yes, but I don't like the idea of having this format in the directory name `2021-02-14T23:00:00Z`. I'd prefer to use underscores `_` instead of `:` or `-`